string mex_id
string job_id
---
bool success